Week3/homework/step4.js

```diff
@@ -1,6 +1,14 @@
 'use strict';
 
 // Add your code here
+function createBase(num1) {
+  const func = function (num2) {
+    let result = 0;
+    result = num1 + num2;
+    console.log(result);
+  };
+  return func;
+}
 
```

Comment:
✅ As a point of style and usefulness when creating higher-order functions.  I would not just print the value out, but also return it from the function.
Otherwise very nicely laid out.